pkg/frontend/admin_openshiftcluster_etcdcertificaterenew.go

```diff
@@ -299,6 +300,44 @@ func (e *etcdrenew) validateEtcdCertsExistsAndExpiry(ctx context.Context) error
 	return nil
 }
 
+func (e *etcdrenew) validateEtcdCertsRenewed(ctx context.Context) error {
```

Comment:
I don't see any unit tests for this new method.

Reply:
Added a new test case. Thanks.

Reply:
The added test cases look good, but I was expecting a new test function for `validateEtcdCertsRenewed` (i.e., `TestValidateEtcdCertsRenewed`, to _just_ test this single method). Are you open to adding that as well?